lib/features/caching/utils/caching_constants.dart

```diff
@@ -19,9 +19,9 @@ class CachingConstants {
   static const int DETAILED_EMAIL_HIVE_CACHE_ID = 17;
 
   static const String fcmCacheBoxName = 'fcm_cache_box';
-  static const String detailedEmailCacheBoxName = 'detailed_email_cache_box';
+  static const String incomingEmailedContentFolderName = 'incoming_emailed';
   static const String newEmailContentFolderName = 'new_email';
```

Comment:
rename to `static const String incomingEmailedContentFolderName = 'incoming_emailed';`